packages/shared/src/hooks/useFeed.ts

```diff
@@ -110,7 +115,7 @@ export default function useFeed<T>(
   const feedQuery = useInfiniteQuery<FeedData>({
     queryKey: feedQueryKey,
     queryFn: async ({ pageParam }) => {
-      const res = await gqlClient2.request(query, {
+      const res = await gqlClient2.request<FeedData>(query, {
```

Comment:
Is this a typo? (the 2?)

Reply:
It's a temp fix to force translations, will be reverted soon

Reply:
It has now been removed, and we only fetch translations when needed.